.github/workflows/js-search.yml

```diff
@@ -0,0 +1,34 @@
+name: JS-Search tests
+
+on:
+  - push
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: "16"
+      - name: Cache node modules
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.rush
+            ~/.pnpm-store
+            common/temp
+            **/node_modules
+          key: ${{ runner.os }}-modules-node16-v1-${{ hashFiles('**/pnpm-lock.yaml') }}
+      - name: Install packages
+        run: node common/scripts/install-run-rush.js update
+      - name: Run tests
+        run: cd pkg/js-search/ && ./node_modules/.bin/jest --ci --reporters=default --reporters=jest-junit
```

Comment:
I would prefer if we had a `test` script in each package's `package.json` (like you've already created for js-search), so that they can be ran using a rush bulk command `rush test`. This would could, but doesn't have to interfere with this workflow: It would make sense to run test on all of the packages using the aforementioned bulk command, or this workflow could stay as it is, running the js-search-specific `rushx test` command.

Reply:
Opened up a ticked related to the matter: #17